arch/platform/openmote/dev/antenna.h

```diff
@@ -32,36 +32,38 @@
  */
 /*--------------------------------------------------------------------------*/
 /**
- * \addtogroup openmote-cc2538
+ * \addtogroup openmote-platform
  * @{
  *
- * \defgroup openmote-antenna OpenMote-CC2538 antenna switch
+ * \defgroup openmote-antenna Antenna switch on OpenMote boards
  *
- * Driver for the OpenMote-CC2538 antenna switch
+ * Driver for the antenna switch on OpenMote platforms.
  * @{
  *
  * \file
- * Header for the OpenMote-CC2538 antenna switch
+ * Header for the OpenMote antenna switch
  */
 /*---------------------------------------------------------------------------*/
 #ifndef ANTENNA_H_
 #define ANTENNA_H_
 /*---------------------------------------------------------------------------*/
 /**
- * \brief Initialize the antenna switch, by default it uses the external
+ * \brief Initialize the antenna switch
  */
 void antenna_init(void);
 /*---------------------------------------------------------------------------*/
 /**
- * \brief Select the external (connector) antenna
- */
-void antenna_internal(void);
-/*---------------------------------------------------------------------------*/
-/**
- * \brief Select the internal (chip) antenna
+ * \brief Function to select the appropriate RF switch configuration.
+ *
+ * \param sel is used to select the desired antenna / radio.
+ *            a value of 0 will connect:
+ *             - OpenMote-CC2538: CC2538 radio to external antenna
+ *             - OpenMote-B     : SMA connector to AT86RF215 radio
+ *            any other value will connect:
+ *             - OpenMote-CC2538: CC2538 radio to internal antenna
+ *             - OpenMote-B     : SMA connector to CC2538 radio
  */
-void antenna_external(void);
-/*---------------------------------------------------------------------------*/
+void antenna_select(uint8_t sel);
```

Comment:
Is there a potential third value for `sel` in the future, or should the parameter be a `bool`?